File: PracticasProgramacion/Practica_14/cpp/Persona.cpp
#include "Persona.h"

Persona::Persona() : nombre_(""), ap_(""), am_(""), genero_(""), edad_(0) {}

Persona::Persona(const std::string& nombre, const std::string& ap, const std::string& am,
                 const std::string& genero, int edad)
    : nombre_(nombre), ap_(ap), am_(am), genero_(genero), edad_(edad) {}

std::string Persona::getNombre() const { return nombre_; }
void Persona::setNombre(const std::string& nombre) { nombre_ = nombre; }
std::string Persona::getAp() const { return ap_; }
std::string Persona::getAm() const { return am_; }
std::string Persona::getGenero() const { return genero_; }
int Persona::getEdad() const { return edad_; }

File: PracticasProgramacion/Practica_14/python/persona.py
"""Persona: nombre, ap, am, genero, edad."""


class Persona:
    def __init__(
        self,
        nombre: str = "",
        ap: str = "",
        am: str = "",
        genero: str = "",
        edad: int = 0,
    ) -> None:
        self._nombre = nombre
        self._ap = ap
        self._am = am
        self._genero = genero
        self._edad = edad

    def get_nombre(self) -> str:
        return self._nombre

    def get_ap(self) -> str:
        return self._ap

    def get_am(self) -> str:
        return self._am

    def get_genero(self) -> str:
        return self._genero

    def get_edad(self) -> int:
        return self._edad
